test/unit/workers/asset_manager_attachment_link_header_update_worker_test.rb

```diff
@@ -11,23 +11,25 @@ class AssetManagerAttachmentLinkHeaderUpdateWorkerTest < ActiveSupport::TestCase
   let(:parent_document_url) { Whitehall.url_maker.public_document_url(edition) }
   let(:update_worker) { mock('asset-manager-update-worker') }
 
-  setup do
-    AssetManagerUpdateAssetWorker.stubs(:new).returns(update_worker)
+  around do |test|
+    AssetManager.stub_const(:AssetUpdater, update_worker) do
+      test.call
+    end
```

Comment:
This is just me being unfamiliar with the testing library. but what does this mean?

Reply:
It means for each test (`test.call`) the `AssetManager::AssetUpdater` constant will instead point to the `update_worker` mock.